src/main/java/ch/njol/skript/expressions/ExprBeehiveFlower.java

```diff
@@ -0,0 +1,71 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.Location;
+import org.bukkit.block.Beehive;
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Beehive Flower Target")
+@Description("The flower a beehive has selected to pollinate from.")
+@Examples({
+	"set the flower target of {_beehive} to block at location(0, 0, 0)",
+	"clear the flower target of {_beehive}"
+})
+@Since("INSERT VERSION")
+public class ExprBeehiveFlower extends SimplePropertyExpression<Block, Location> {
+
+	static {
+		registerDefault(ExprBeehiveFlower.class, Location.class, "flower target", "blocks");
```

Comment:
```suggestion
		registerDefault(ExprBeehiveFlower.class, Location.class, "target flower", "blocks");
```
Maybe ExprTarget?

Reply:
Do you think I should? I don't really wanna convolute it anymore than it already it is.
Let me know

Reply:
probably not, actually, target block of block just sounds silly